### ISSUE TITLE
DoK essence lash hitting over 3k

### ISSUE THREAD

**Issue by SunsetCNTK** [WarEmu/WarBugs#14334]
Issues should be unique. Check if someone else reported 
the issue first, and please don't report duplicates.
Make sure you've checked the wiki before you post something ToK related:
https://wiki.returnofreckoning.com/

**Expected behavior and actual behavior:**
It sure shouldn't hit for over 3k

**Steps to reproduce the problem:**
~~Get hit by a DoK I suppose
Or log on DoK and hit someone~~

May be some specific tactic that causes it to deal so much dmg, will experiment with my dok once my guild session is over

**Testing Screenshots/Videos/Evidences (always needed):**
![Sunset_039](https://user-images.githubusercontent.com/16652212/58827693-e5551500-8643-11e9-8add-2473cddddd42.jpg)

**Comment by SunsetCNTK** [comment]
May be some specific tactic that causes it to deal so much dmg, will experiment with my dok once my guild session is over

**Comment by sioding** [comment]
Tried also to reproduce it and can't but we also see that happen.

**Comment by Parallell86** [comment]
Same DOK hit me with an auto attack for 4.3k, 6.4k mitigated. Ergo, almost 11k damage.

**Comment by ElMosquito** [comment]
FYI relogging removed whatever bug it was and the numbers are back to normal.

**Comment by grunbag** [comment]
Is it fixed ?

**Comment by Parallell86** [comment]
No i dont think its a bug mate. Because it wasnt doks in general. Just this one.

He was cheating his ass off that night.

Skaffa Outlook för Android<https://aka.ms/ghei36>

________________________________

**Comment by ElMosquito** [comment]
It was fixed the moment I relogged. Just like that SW with 65k ranged power in the other thread, I think I got that same bug when logging in. I'm not sure if it only affected damage and not healing in my case (as a healer in wb I didn't notice anything special about my healing so it most likely was only melee power that was affected). Not entirely sure how stats are scaling with character level, if it is connected then perhaps it got wrong somehow and scaled to the character's renown level instead when logging in.
